frontend/src/credit_transfers/components/CreditTransferFormButtons.js

```diff
@@ -66,22 +72,28 @@ const CreditTransferFormButtons = props => (
       </TooltipWhenDisabled>
       }
       {props.actions.includes(Lang.BTN_REFUSE) &&
-      <button
-        className="btn btn-danger"
-        data-target="#confirmRefuse"
-        data-toggle="modal"
-        type="button"
+      <TooltipWhenDisabled
+        disabled={props.isCommenting}
+        title={Lang.TEXT_COMMENT_DIRTY}
       >
-        {Lang.BTN_REFUSE}
-      </button>
+        <button
+          className="btn btn-danger"
+          disabled={props.isCommenting}
+          data-target="#confirmRefuse"
+          data-toggle="modal"
+          type="button"
+        >
+          {Lang.BTN_REFUSE}
+        </button>
+      </TooltipWhenDisabled>
       }
       {props.actions.includes(Lang.BTN_SIGN_2_2) &&
       <TooltipWhenDisabled
-        disabled={props.disabled.BTN_SIGN_2_2}
-        title={props.permissions.BTN_SIGN_2_2
+        disabled={props.isCommenting || props.disabled.BTN_SIGN_2_2}
+        title={props.isCommenting ? Lang.TEXT_COMMENT_DIRTY : (props.permissions.BTN_SIGN_2_2
```

Comment:
I know that eslint will complain about the nested ternary, but it's the most concise way to do this, and the parens make precedence unambiguous.